### DISCUSSION TITLE
bulk creation of contact associations

### DISCUSSION THREAD

**Discussion by bgoulet00** [nautobot/nautobot#6477] [Q&A] [answered]
is there a way to create contact associations to locations using csv import?

**Comment by bgoulet00** [comment]
can this only be done programmatically?

**Comment by glennmatthews** [comment, marked as answer]
Should be doable by directly accessing the "Import Objects" job and selecting `extras | contact association` as the content type:

![image](https://github.com/user-attachments/assets/d302d7a1-f810-409d-85dc-d7a5ec79e76f)

**Reply by bgoulet00** [reply]
trying to associate contacts to locations.....
i assume `associated_object_id` is the UUID of the location object.  must be done by UUID and not by name
i'm not sure what it is looking for with `associated_object_type` in the format `app_label.model`
i've unsuccessfully tried several variations of `dcim_location` and `dcim_locationtype`
after some searching this looks like it might be some django thing?  i have no experience with django so i'm not sure how to approach this

**Reply by glennmatthews** [reply]
`associated_object_type` would be `dcim.location` or `dcim.locationtype`. You are correct about `associated_object_id`.

**Reply by bgoulet00** [reply]
perfect! thank you.  BTW, just ordered your book online so hopefully that will mean less noob questions out of me.  :)
